src/main/kotlin/org/jqassistant/tooling/intellij/plugin/common/PluginUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.jqassistant.tooling.intellij.plugin.common
+
+import com.intellij.openapi.vfs.JarFileSystem
+import com.intellij.openapi.vfs.VfsUtil
+import com.intellij.openapi.vfs.VirtualFile
+import org.jqassistant.schema.plugin.v2.JqassistantPlugin
+import org.jqassistant.tooling.intellij.plugin.data.plugin.JqaPlugin
+import java.io.File
+import kotlin.io.path.Path
+
+object PluginUtil {
+    private const val PLUGIN_FILE_PATH = "META-INF/jqassistant-plugin.xml"
+    private const val PLUGIN_RULE_PATH = "META-INF/jqassistant-rules"
```

Comment:
```suggestion
    private const val PLUGIN_FILE_PATH = PluginConfigurationReader.PLUGIN_RESOURCE
    private const val PLUGIN_RULE_PATH = ClasspathRuleSource.RULE_RESOURCE_PATH
```

Maybe instead of hardcoding the paths we could use the ones defined in jqassistant from
```kotlin
import com.buschmais.jqassistant.core.plugin.api.PluginConfigurationReader
import com.buschmais.jqassistant.core.rule.api.source.ClasspathRuleSource
```